shakenfist/client/main.py

```diff
@@ -597,7 +597,7 @@ def instance_list(ctx, all=False):
 def _pretty_data(row, space_rules):
     ret = ''
     for key in space_rules:
-        ret += key + '=' + str(row[key]).ljust(space_rules[key]) + '  '
+        ret += key + '=' + str(row.get(key, '')).ljust(space_rules[key]) + '  '
```

Comment:
The get() was to enable use of the function when the data row didn't contain the entire set of keys in space_rules.
It turns out we don't need that at the moment.
So, yes. I guess that was cruft...